packages/component/src/BasicTranscript.js

```diff
@@ -15,20 +16,23 @@ import removeInline from './Utils/removeInline';
 import ScreenReaderActivity from './ScreenReaderActivity';
 import ScrollToEndButton from './Activity/ScrollToEndButton';
 import SpeakActivity from './Activity/Speak';
-import useActivities from './hooks/useActivities';
-import useCreateActivityRenderer from './hooks/useCreateActivityRenderer';
-import useCreateActivityStatusRenderer from './hooks/useCreateActivityStatusRenderer';
-import useCreateAvatarRenderer from './hooks/useCreateAvatarRenderer';
-import useDirection from './hooks/useDirection';
 import useFocus from './hooks/useFocus';
-import useGroupActivities from './hooks/useGroupActivities';
-import useLocalizer from './hooks/useLocalizer';
 import useMemoize from './hooks/internal/useMemoize';
-import useStyleOptions from './hooks/useStyleOptions';
 import useStyleSet from './hooks/useStyleSet';
```

Comment:
expand from `hooks`?

Reply:
The hooks are inside the same package, so we are not importing from `hook` then expanding it again. But good call.